### ISSUE TITLE
sample csv file

### ISSUE THREAD

**Issue by rockneverdies55** [owenbean400/fastlane-plugin-google_data_safety#1]
Would you please provide a sample .csv file that the plugin expects?

**Comment by owenbean400** [comment]
On the Google Play Console, go to the menu: “Monitor and improve” > “Policy and programs” > “App Content” > “Data Safety.” There, you will find an option to export a CSV file. For my work, I used this file and the Google API to upload the data safety information to all of our other apps. I decided to turn the API upload to a Fastlane plugin.

Secondly, there is a template file provided by Google, which can be downloaded here: [Google Play Support Data Safety](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en). I have also attached the template file for your convenience:
[csv_template.csv](https://github.com/user-attachments/files/18136309/csv_template.csv).

Lastly, I am currently working on a command-line prompt with yes-or-no questions about app data. This tool will generate a completed CSV file ready for upload.
